quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/prereqs/prereq.azuredeploy.json

```diff
@@ -35,18 +35,6 @@
         "description": "The mobile network code for the private mobile network"
       }
     },
-    "sliceName": {
```

Comment:
Why the slice and sst parameters are removed from prereq Mobile Network?

Reply:
we want the quickstart templates to be a fast way to setup a basic mobile network -we're considering multiple slices/ non-default slice as non-basic